svir/dialogs/viewer_dock.py

```diff
@@ -373,6 +373,24 @@ def filter_dmg_by_asset_aggr(self):
             return
         self.draw_dmg_by_asset_aggr()
 
+    def filter_agg_curves(self):
+        params = {}
+        for tag_name in self.tags:
+            if self.tags[tag_name]['selected']:
+                for value in self.tags[tag_name]['values']:
+                    if self.tags[tag_name]['values'][value]:
+                        # NOTE: this would not work for multiple values per tag
```

Comment:
It would be useful in a comment to write down the structure of the .tags dictionary in the example of the demo.

Reply:
`self.tags` contains a lot of tags and corresponding values. It's too big to be written in a comment here. I can edit it and reduce it to a very small subset, in order to make it readable here.